test/Microsoft.AspNet.Mvc.IntegrationTests/BinderTypeBasedModelBinderIntegrationTest.cs

```diff
@@ -163,7 +163,7 @@ private class Person2
             Assert.True(modelState.IsValid);
             var key = Assert.Single(modelState.Keys);
             Assert.Equal("CustomParameter", key);
-            Assert.Null(modelState[key].Value); // value is only set if the custom model binder sets it.
+            Assert.NotNull(modelState[key].Value); //Do not validate as its the test model binder which sets the value.
```

Comment:
If there is a key set here can we perform any stronger validation about what's stored? Values/Errors/etc?